### DISCUSSION TITLE
Flask 文档中文翻译计划

### DISCUSSION THREAD

**Discussion by greyli** [greyli/helloflask#239] [General]
如果你感兴趣想要参与的话，欢迎订阅这个讨论（右侧 Subscribe 按钮），后续动态会更新到这里（留言后也会自动订阅）。谢谢！关于翻译协作有什么想法和建议也欢迎讨论。

**Comment by zzmacoolboy** [comment]
感兴趣，希望能订阅，谢谢！

**Comment by ohperhaps** [comment]
非常棒

**Comment by greyli** [comment]
翻译项目已创建： https://github.com/greyli/flask-docs-zh

欢迎参与！
